app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -330,5 +324,22 @@ def activation_key_params
 
       key_params
     end
+
+    def parent_scope
```

Comment:
The Foreman auth model enables authorization to look up into the parent scope(s). Things didn't quite work when doing that, so to keep with the current way of not looking into the parent scope we can override this method. Can move this to a parent controller, but this is for demonstration.